pkg/asset/k8s.go

```diff
@@ -40,6 +41,9 @@ func newStaticAssets(imageVersions ImageVersions) Assets {
 		MustCreateAssetFromTemplate(AssetPathControllerManagerDisruption, internal.ControllerManagerDisruptionTemplate, conf),
 		MustCreateAssetFromTemplate(AssetPathKubeDNSDeployment, internal.DNSDeploymentTemplate, conf),
 		MustCreateAssetFromTemplate(AssetPathCheckpointer, internal.CheckpointerTemplate, conf),
+		Asset{AssetPathCheckpointerSA, internal.CheckpointerServiceAccount},
```

Comment:
I think it might be more uniform (and less surprising in the future) to pass these through `MustCreateAssetFromTemplate` even if they currently do not do any templating.